tools/mo/openvino/tools/mo/front/mxnet/arange_like_replacer.py

```diff
@@ -0,0 +1,148 @@
+                                                                      2: int64_array(0)},
+                                                      {'name': name + '/Gather'})
+            input_shape_node.out_port(0).connect(gather_node.in_port(0))
+            gather_node.out_port(0).connect(range_node.in_port(1))
+            node.out_port(0).get_connection().set_source(range_node.out_port(0))
+            rename_nodes([(node, name + '/ShouldBeDeleted'), (range_node, name)])
+        else:
+            r'''
+            Replace arange_like op to subgraph:
+                    |
+                    | -------------- |
+                    |                |
+                 Reshape             |
+                    |                 |
+                 ShapeOf           ShapeOf  
+                    |                |
+                  Range              |
+                    |                |
+                 Reshape ----------- | 
+                    |
+            '''
+
+            reshape_forward_node = create_op_with_const_inputs(graph, Reshape, {1: int64_array([-1])},
+                                                               {'name': name + '/Reshape_forward'})
+
+            reshape_backward_node = Reshape(graph, {'name': name + '/Reshape_backward'}).create_node()
+            shape_of_node = Shape(graph, {'name': reshape_forward_node.name + '/ShapeOf'}).create_node()
+
+            reshape_forward_node.in_port(0).connect(data_out_port)
+            reshape_forward_node.out_port(0).connect(shape_of_node.in_port(0))
+            shape_of_node.out_port(0).connect(range_node.in_port(1))
+            range_node.out_port(0).connect(reshape_backward_node.in_port(0))
+            input_shape_node.out_port(0).connect(reshape_backward_node.in_port(1))
+            node.out_port(0).get_connection().set_source(reshape_backward_node.out_port(0))
+            rename_nodes([(node, name + '/ShouldBeDeleted'), (reshape_backward_node, name)])
+
+        if node.repeat != 1:
+            r"""
+            First, we generate the correct stop value for Range like new_stop_value = stop_value // repeat + 1.
+            Then repeats each value of the interval using Tile. After that we can get a longer interval
+            so we reduce it with Slice.
+            
+            Sub-graph after Range node will be look like
+            
+            Range - Reshape([-1, 1]) - Tile([1, repeat]) - Reshape(-1) - Slice
+            
+            """
+            div_node = create_op_with_const_inputs(graph, Div, {1: int64_array([node.repeat])},
+                                                   {'name': name + '/Divide'})
+            add_node = create_op_with_const_inputs(graph, Add, {1: int64_array([1])},
+                                                   {'name': div_node.name + '/Add'})
+            cast_node = Cast(graph, {'name': name + '/ConvertToI64', 'dst_type': np.int64}).create_node()
```

Comment:
nit: this is not necessary but if you have enough time please add this node into subgraph picture in line 50

Reply:
This node does not belong to the subgraph in line 50